app/src/main/scala/im/tox/antox/callbacks/AntoxOnFileReceiveCallback.scala

```diff
@@ -40,10 +40,14 @@ class AntoxOnFileReceiveCallback(ctx: Context) extends FileReceiveCallback {
       }
 
       val fileId = ToxSingleton.tox.fileGetFileId(friendNumber, fileNumber).toString
-      val storedFileId = ToxSingleton.tox.hash(AVATAR.getAvatarFile(name, ctx).orNull).orNull
```

Comment:
Avatars weren't working for me because of this line. It would get here, then after executing tox.hash() it would just return and go back to toxcore iterations(). By separating it out like I've done, it seems to work as normal and compares the hashes to see if a download is needed